[2025-03-19T18:42:17.794+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T18:42:17.895+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.greet scheduled__2024-01-11T00:00:00+00:00 [queued]>
[2025-03-19T18:42:17.947+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.greet scheduled__2024-01-11T00:00:00+00:00 [queued]>
[2025-03-19T18:42:17.948+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T18:42:18.115+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): greet> on 2024-01-11 00:00:00+00:00
[2025-03-19T18:42:18.177+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1492) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T18:42:18.187+0000] {standard_task_runner.py:72} INFO - Started process 1513 to run task
[2025-03-19T18:42:18.199+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v1', 'greet', 'scheduled__2024-01-11T00:00:00+00:00', '--job-id', '339', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpbns87je3']
[2025-03-19T18:42:18.219+0000] {standard_task_runner.py:105} INFO - Job 339: Subtask greet
[2025-03-19T18:42:18.456+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v1.greet scheduled__2024-01-11T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T18:42:18.746+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v1' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2024-01-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-11T00:00:00+00:00'
[2025-03-19T18:42:18.747+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T18:42:18.753+0000] {logging_mixin.py:190} INFO - Hello Jerry, you are 19 years old.
[2025-03-19T18:42:18.754+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-19T18:42:18.765+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T18:42:18.767+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v1, task_id=greet, run_id=scheduled__2024-01-11T00:00:00+00:00, execution_date=20240111T000000, start_date=20250319T184217, end_date=20250319T184218
[2025-03-19T18:42:18.887+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T18:42:18.931+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-19T18:42:18.935+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
